### ISSUE TITLE
4T

### ISSUE THREAD

**Issue by bsopilnyak-eng** [bsopilnyak-eng/ProjectIAD_BS_PB#4]
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from sklearn.linear_model import LinearRegression

data = pd.DataFrame({
    'Глюкоза': [4.0, 4.5, 5.0, 5.5, 6.0, 6.5, 7.0, 7.5, 8.0, 8.5],
    'Гемоглобін': [12.5, 12.8, 13.2, 13.6, 14.0, 14.5, 15.0, 15.4, 15.7, 16.0]
})

X = data[['Глюкоза']]
y = data['Гемоглобін']

model = LinearRegression()
model.fit(X, y)

plt.scatter(X, y, label='Дані')
plt.plot(X, model.predict(X), color='red', label='Регресійна лінія')
plt.xlabel("Глюкоза")
plt.ylabel("Гемоглобін")
plt.legend()
plt.show()

print("Коефіцієнт детермінації R²:", model.score(X, y))
print("Рівняння регресії: y = %.3f * x + %.3f" % (model.coef_[0], model.intercept_))
